docs/Q-and-A.md

```diff
@@ -0,0 +1,635 @@
+See the [definitions](#normative) section for what both are. For example, theories of ethics are generally `normative` - you should not kill, you should help that person, etc. Economics is most commonly `non-normative` - instead of asking “how should this person choose which goods to buy?”, we are often more interested in “how does this person choose which commodities they buy?”.
+
+## **Q: Is Sidetree privacy preserving / GPDR proof?
+These are the information-hiding measures taken in the protocol to facilitate compliance with privacy rules:
+- No sensitive data on layer 1, which can be an open public blockchain
+- THe content in the CAS layer 2, example IPFS, works with `commitments`. A commitment can be shared without revealing the commit value.
+- Files-structures are designed to minimize permanently retained data
+- Operation Request of type Recover or Deactivate
+{TBW prio 2: unclear wether `Deactivate` obeys the right to be forgotten}\
+_(@henkvancann)_
+
+## **Q: Is there like a central, GitHub repository that someone or some group of people are responsible for?
+_Could we contribute to the repository?_
+
+it is in the Decentralized Identity Foundation's GitHub depository, [Sidetree](https://github.com/decentralized-identity/sidetree) is the name of it. So you can contribute code there, you can read every line of code. There's no code insertions after the fact, like every bit of every piece of code that we run, as an ION node, is there. So it's totally inspectable.
+- [Contribution guidelines for the project](https://github.com/decentralized-identity/sidetree/blob/master/docs/contribution-guidlines.md).
+- [The Sidetree release process](https://github.com/decentralized-identity/sidetree/blob/master/docs/release-process.md)
+- [The Sidetree coding style guide](https://github.com/decentralized-identity/sidetree/blob/master/docs/styleguide.md)
+
+# Q&A section Sidetree operational
+
+## *Q:  Where can I download Sidetree?
+On (sub)page(s of) [github](https://github.com/decentralized-identity/sidetree)
+
+##  *Q: Where can we find the code and how could a coder get started?
+The homepage on github [README.md](../README.md) pretty much sums up all the possibilities to download the available code and how developers can engage in the development process currently. We welcome contributions.
+
+##  **Q: What are the external dependencies of Sidetree?
+_I hear ya: "Sidetree is able to do all this **without requiring** trusted intermediaries, centralized authorities, special protocol tokens, or secondary consensus mechanisms, while preserving the core attributes of decentralization and immutability of the underlying ledger systems it is implemented on." But what does Sidetree require instead?_
+
+Sidetree is dependent of:
+1. Proper IPFS data management and Identity and Access Management via API keys?
+2. Reasonable fees and confirmation-times on layer 1
+3. Mining and governance of the layer 1 blockchain it settles on.
```

Comment:
An anchoring system - mining is not applicable to all anchoring systems.

Reply:
True, thx, changed: 
Consensus, transaction validation and governance in general of the layer 1 blockchain it settles on.